recipes/pymc3_hmm/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pymc3-hmm" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9a4d89776d34ffe16fdb5450b00957856d06d24253d7f2a4a39dd1aad8ba36cd
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
```

Comment:
Looks like this needs `pymc3`, at least.

Reply:
Updated. However, been getting [this dependency error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=279970&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=755) while running docker build. I basically copied the version dependencies directly from the `pymc3-hmm` repo. Do we need to adjust some of these?